src/styles/main.scss

```diff
@@ -1,7 +1,16 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap");
```

Comment:
Consider importing the Google Fonts in your HTML file within the head tag, instead of importing it directly in the SCSS file.